assets/src/edit-story/app/snackbar/snackbarContainer.js

```diff
@@ -47,7 +47,7 @@ const Container = styled.div`
   top: ${({ place }) => (place.indexOf('top') === 0 ? 0 : 'inherit')};
   bottom: ${({ place }) => (place.indexOf('bottom') === 0 ? 0 : 'inherit')};
   left: ${({ place }) => getLeft(place)};
-  z-index: 10000;
+  z-index: 2147483647;
```

Comment:
That's ... odd. Why is it such an  arbitrary number and what does it solve? At least a comment would be nice.

NB: We should really define our `z-index` usage in `theme.js`

Reply:
It's not really arbitrary, but it definitely looks like one. `2147483647` is the maximum possible zIndex value. But I don't believe z-indexes belong in `theme.js` - they have nothing to do with themes and do not change when a theme changes. I'd be very happy, though, if we put global zIndex values in their own `zindex.js` file. But I'd stress the global aspect - basically this is only relevant for `position:fixed` elements. Other uses of `z-index` are localized to a stacking context and would be more confusing than helpful in a global const file.

Reply:
Oh I see, 2^31 🤦 . Yes, a separate `zindex.js` file would be nice in the future.